fs/dafsclient/rmtfile.cpp

```diff
@@ -1349,6 +1349,8 @@ class CRemoteFileIO : public CInterfaceOf<IFileIO>
             return ioWrites.load(std::memory_order_relaxed);
         case StNumDiskRetries:
             return ioRetries.load(std::memory_order_relaxed);
+        case StTimeLocalExecute:
+            return cycle_to_nanosec(ioReadCycles.load(std::memory_order_relaxed)+ioWriteCycles.load(std::memory_order_relaxed));
```

Comment:
looks like a correct addition/change, but not directly related to this JIRA?

Reply:
The disk join is using the diskReadActivityStatistics and this statistics mapping  has StTimeLocalExecute in its list of stats.